historical_system_profiles/models.py

```diff
@@ -0,0 +1,40 @@
+import uuid
+from datetime import datetime
+
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+db = SQLAlchemy()
+
+
+class HistoricalSystemProfile(db.Model):
+    __tablename__ = "historical_system_profiles"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    account = db.Column(db.String(10))
+    inventory_id = db.Column(UUID(as_uuid=True))
+    created_on = db.Column(db.DateTime, default=datetime.utcnow)
+    modified_on = db.Column(
+        db.DateTime, default=datetime.utcnow, onupdate=datetime.utcnow
+    )
+    system_profile = db.Column(JSONB)
```

Comment:
Should we store the sync time from the profile as a top level attribute on the model itself to make fetching and filtering easier?

Reply:
I think you can query json fields via postgres somehow, instead of pulling the whole thing and parsing. I'll give it a shot.